### PR TITLE
add(ci): spellcheck action

### PR DESCRIPTION
**Pull request by TomaszWaszczyk**
WIP

### CODE REVIEW

**Comment by pepoviola** [comment]
Looks good, we may need to add an extra dictionary.
Thx!

**Comment by TomaszWaszczyk** [comment]
I have added custom dictionary, on my end after execution of `cargo spellcheck` I have empty console.

**Review by pepoviola — APPROVED** [review]

**Comment by pepoviola** [comment]
Looks good @TomaszWaszczyk, thanks!!

**Comment by TomaszWaszczyk** [comment]
@pepoviola Is there anything else I can do to help merge the pull request?

**Review by pepoviola — APPROVED** [review]

**Comment by pepoviola** [comment]
> @pepoviola Is there anything else I can do to help merge the pull request?

Sorry about the delay, merging now. Thanks!!

File: .github/workflows/spellcheck.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        uses: actions/checkout@v3
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+
+      - name: Install cargo-spellcheck
+        run: cargo install cargo-spellcheck
```
Comment (pepoviola) [inline .github/workflows/spellcheck.yml:26]:
```suggestion
        run: |
           apt-get install libclang-dev
           export LIBCLANG_PATH=/usr/lib/llvm-14/lib/
           cargo install cargo-spellcheck
```

File: .github/workflows/spellcheck.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
+
+      - name: Install cargo-spellcheck
+        run: |
+          apt-get install libclang-dev
```
Comment (pepoviola) [inline .github/workflows/spellcheck.yml:27]:
```suggestion
          sudo apt-get install libclang-dev
```

**Comment by pepoviola** [comment]
Fixed needed deps, mergin now.
Thx @TomaszWaszczyk!

**Pull request merged**